lib/tests/lib.rs

```diff
@@ -0,0 +1,66 @@
+use fvm_shared::error::ExitCode;
+use kythera_common::abi::{Abi, Method, MethodType};
+use kythera_lib::{TestResultType, Tester, WasmActor};
+use kythera_test_actors::wasm_bin::BUILTIN_TEST_ACTOR_BINARY;
+
+const TARGET_WAT: &str = r#"
+        ;; Mock invoke function
+            (module
+                (func (export "invoke") (param $x i32) (result i32)
+                    (i32.const 1)
+                )
+            )
+        "#;
+
+#[test]
+fn test_builtin_deployed() {
```

Comment:
great test. thanks for that.

should we work a bit on code duplication between this and tests in `lib/src/lib.rs` before moving forward?